motionplan/motionPlanner.go

```diff
@@ -463,21 +462,21 @@ func CheckPlan(
 	plan Plan,
 	worldState *frame.WorldState,
 	fs frame.FrameSystem,
-	currentPosition spatialmath.Pose,
-	currentInputs []frame.Input,
+	currentPose spatialmath.Pose,
+	currentInputs map[string][]frame.Input,
```

Comment:
[q/opt] Should this be renamed to currentTrajectoryInputs, or something similar?

Reply:
zeroTrajectoryInputs?